test/integration/users.test.js

```diff
@@ -679,12 +679,12 @@ describe("Users", function () {
       expect(res.body.message).to.equal("Route not found");
     });
 
-    it("Should return an error when passing an invalid Discord ID", async function () {
+    it("Should return an empty object when passing an invalid Discord ID", async function () {
       const invalidDiscordId = "50485556209423";
       const res = await chai.request(app).get(`/users?dev=true&discordId=${invalidDiscordId}`);
-      expect(res).to.have.status(404);
+      expect(res).to.have.status(200);
```

Comment:
why there is a success status of 200 if a discord ID is wrong?

Reply:
This was changed as per discussion with @iamitprakash  and @ankushdharkar.

Reply:
But here you are testing for the wrong ID then it should be 404